src/components/sections/general/form-sections/button-field.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import i18n from '@dhis2/d2-i18n'
+import PropTypes from '@dhis2/prop-types'
+import formStyles from '../../../../styles/Form.module.css'
+import { Button, Field } from '@dhis2/ui'
+
+export const ButtonField = ({
+    handleForm,
+    handleDisableSettings,
+    syncElement,
+}) => {
+    const { label, helpText } = syncElement
+
+    return (
+        <div className={formStyles.rowMargin}>
+            <Field helpText={i18n.t('{{helpText}}', { helpText })}>
+                <Button
+                    onClick={handleDisableSettings.open}
+                    disabled={handleForm.fields.disableAll}
+                >
+                    {i18n.t('{{label}}', { label })}
```

Comment:
Calling `i18n.t` here with only a variable doesn't do anything, the text needs to be translated before it is passed to this component.